### PR TITLE
Recall associations

### PR DESCRIPTION
**Pull request by sarikasu**

### CODE REVIEW

File: tasks/recall-association.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Summary
+> This study was designed to understand collaborative inhibition causes. Researchers compare individual recall (control) to two collaborative group conditions, 
+one in which people took turns recalling items together (See Group), the other in which people take turns recalling items but are not aware of the items their partner 
+has recalled (No See Group). Findings how that collaborative inhibition is due to interference caused by the product of recall- sharing answers with each other,
```
Comment (katelynbottcher) [inline tasks/recall-association.md:4]: show*

File: tasks/recall-association.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+> Productivity Loss In Brainstorming Groups: Toward the Solution of a Riddle
+September 1987 Journal of Personality and Social Psychology 53(3):497-509
+DOI: 10.1037/0022-3514.53.3.497: https://www.researchgate.net/publication/232602288_Productivity_Loss_In_Brainstorming_Groups_Toward_the_Solution_of_a_Riddle
+
+> Group remembering: does social loafing underlie collaborative inhibition? M S Weldon 1, C Blair, P D Huebsch
+PMID: 11185783 DOI: 10.1037//0278-7393.26.6.1568
+
+
+# Stimuli
+> No Visuals Included in paper or appendix but booklet with words printed in black 20 point font was used
```
Comment (katelynbottcher) [inline tasks/recall-association.md:20]: add, "with each word printed on a separate sheet of paper, displayed in the center"

Comment (katelynbottcher) [reply tasks/recall-association.md:20]: I see you added it later

File: tasks/recall-association.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+> Step One: Participants were randomly assigned to one of three conditions with the restriction that there were 20 people in each condition 
+        Nominal Condition: participants would work alone to recall words: Tested individually 
+        Collaborative Group See Condition: participants worked in pairs and took turns recalling words and were showed the words that 
+        each other were recalling. They sat opposite each other at a table
+        Collaborative Group No See Condition: participants recalled word lists in pairs, using a turn-taking method however they did not 
+        see what word the other recalled
+##
+> Step Two: Participants are told they will see a list of words that will be tested on later but not told how they will be tested
+##
+> Step Three: Participants are given a booklet containing a word list, they are told to start on page one and study each word (one word in the middle of each page in 20 point black font). They are not allowed to talk or write anything down during the study period
```
Comment (katelynbottcher) [inline tasks/recall-association.md:36]: studying each word for 4 seconds

File: tasks/recall-association.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+PMID: 11185783 DOI: 10.1037//0278-7393.26.6.1568
+
+
+# Stimuli
+> No Visuals Included in paper or appendix but booklet with words printed in black 20 point font was used 
+
+## Materials for alternative versions of the experiment 
+> N/A
+
+# Procedure
```
Comment (katelynbottcher) [inline tasks/recall-association.md:25]: I actually feel like the procedure should be a more general outline, and the instructions should include the details about what participants are actually told, as seen in this example https://docs.google.com/document/d/1yxQfHobL3vAamNgnt14korvORaI-N1ed62gHsEovHXI/edit

File: tasks/recall-association.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        Collaborative Group See Condition: participants worked in pairs and took turns recalling words and were showed the words that 
+        each other were recalling. They sat opposite each other at a table
+        Collaborative Group No See Condition: participants recalled word lists in pairs, using a turn-taking method however they did not 
+        see what word the other recalled
+##
+> Step Two: Participants are told they will see a list of words that will be tested on later but not told how they will be tested
+##
+> Step Three: Participants are given a booklet containing a word list, they are told to start on page one and study each word (one word in the middle of each page in 20 point black font). They are not allowed to talk or write anything down during the study period 
+##
+> Step Four: After study period, participants completed addition problems for sixty seconds to reduce recency effects
```
Comment (katelynbottcher) [inline tasks/recall-association.md:38]: Include recall instructions for each group

File: tasks/recall-association.md (hunk 4 above)
Comment (sarikasu) [reply tasks/recall-association.md:25]: @katelynbottcher I took a stab at reformatting, does this feel more on target now?

**Review by katelynbottcher — APPROVED** [review]

**Pull request merged**